src/components/ui/github.tsx

```
@@ -1,15 +1,20 @@
+  const { resolvedTheme } = useTheme();
+
+  return (
+    <svg
+      viewBox="0 0 256 250"
+      width="1em"
+      height="1em"
+      fill={resolvedTheme === "dark" ? "#fff" : "#24292f"}
+      xmlns="http://www.w3.org/2000/svg"
+      preserveAspectRatio="xMidYMid"
+      {...props}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)